assets/js/pages/Users/UserForm.jsx

```diff
@@ -58,6 +69,9 @@ function UserForm({
   const [confirmPasswordState, setConfirmPassword] = useState('');
   const [confirmPasswordErrorState, setConfirmPasswordError] = useState(null);
   const [statusState, setStatus] = useState(status);
+  const [selectedAbilities, setAbilities] = useState(
+    userAbilities.map(({ id }) => id)
```

Comment:
Wouldn't be something like "effectiveAbilities" more appropriate? I'm asking because this seems to reflect the abilities that are effective at a certain moment and not the ones that have been selected in terms of UI, but not yet submitted. (Hope i'm not misunderstanding)

Reply:
`selectedAbilities` indeed reflects the abilities that are selected for that user in that moment in the UI.
Initially, the abilities that are currently stored in the DB of course.
When you change them using the UI, the values are changed.
If they are submitted or not is irrelevant for the state.

Reply:
And if you mean the `userAbilities`, this are the initial user abilities.
It could be named with many other ways yes, like `currentAbilities`, `userCurrentAbilities`, etc
I think it is quite explicit what it means